src/Playroom/Snippets/Snippets.tsx

```diff
@@ -142,9 +143,15 @@ export default ({ snippets, onHighlight, onClose }: Props) => {
                 [styles.highlight]: isHighlighted,
               })}
               onMouseMove={
-                isHighlighted ? undefined : () => setHighlightedIndex(index)
+                isOpen && !isHighlighted
+                  ? () => {
+                      setHighlightedIndex(index);
+                    }
+                  : undefined
               }
-              onMouseDown={() => closeHandler(filteredSnippets[index])}
+              onMouseDown={() => {
+                closeHandler(filteredSnippets[index]);
+              }}
```

Comment:
Could make this diff smaller by removing the curly braces from the functions. Github wouldn't let me suggest a change :(